File: Iniciante/1070 - Seis Números Ímpares/Python/1070.py
def main():
    X = int(input())

    valores_impares(X)


def valores_impares(x):
    i = 0

    while i < 6:
        if x % 2 == 0:
            x += 1
        print(x)
        x += 2
        i += 1


main()

File: Iniciante/1070 - Seis Números Ímpares/JavaScript/1070.js
function main() {
  var input = require("fs").readFileSync("/dev/stdin", "utf8");
  const X = parseInt(input);

  valoresImpares(X);
}

function valoresImpares(x) {
  for (let i = 0; i < 6; i++) {
    if (x % 2 === 0) {
      x += 1;
    }
    console.log(x);
    x += 2;
  }
}

main();
